.github/workflows/issue_checks.yml

```
@@ -11,7 +11,7 @@ permissions:
 
 jobs:
   script_check:
-    name: Script Check
+    name: Closed Issue Check
```

```suggestion
    name: Closed TODO Issue Check
```

(Let's try to be as specific as we can in this context).

---

Done.